src/main/java/FileWriterService.java

```diff
@@ -6,8 +6,8 @@
 
     public void writeLinesToFile(String fileName, String report) {
         try {
-            Files.write(Paths.get("report.txt"), report.getBytes());
-        } catch (IOException e) {
+            Files.write(Paths.get(fileName), report.getBytes());
+        } catch (Exception e) {

```

Comment:
Maybe it's better to catch specific exceptions, e.g. FileNotFoundException & IOException, so that we can later see in logs what the problem was

Reply:
Done